custom/enikshay/two_b_datamigration/management/commands/import_drtb_cases.py

```diff
@@ -861,50 +1152,89 @@ def clean_phone_number(value, digits):
     if not value:
         return None
     assert digits in (10, 12)
-    exception = Exception("Unexpected phone number format: {}".format(value))
-    cleaned = re.sub('[^0-9]', '', value)
+
+    # TODO: (WAITING) what to do if there are two numbers?
+    try:
+        values = value.split("/")
+        value = values[0]
+    except AttributeError:
+        # This exception will be raised if value is an int.
+        pass
+
+    cleaned = re.sub('[^0-9]', '', str(value))
```

Comment:
@NoahCarnahan if the cleaned number is 11 digits long and starts with a leading zero we can just remove the leading zero